sources/meta-srobo/recipes-images/images/srobo-image-robot.bb

```
@@ -20,6 +20,9 @@ CORE_IMAGE_EXTRA_INSTALL += " \
     python3-pandas \
     python3-pillow \
     python3-pyudev \
+    python3-pip \
```

```suggestion
    python3-pip \
    python3-wheel \
```

We should provide wheel support so that `.whl` files can be downloaded from PyPI. This means that they don't need to sort out Internet access on the Pi (i.e locked down school internet)